cmd/gokr-packer/packer.go

```diff
@@ -423,17 +457,48 @@ func logic() error {
 
```

Comment:
Above this line, in the `if *update == "yes"` check, please construct the URL with `https` when `*useTLS` is non-empty